bench/bench.gradle.kts

```diff
@@ -10,7 +10,7 @@ val graal: Configuration by configurations.creating
 
 @Suppress("UnstableApiUsage")
 dependencies {
-  jmh(project(":pkl-core"))
+  jmh(projects.pklCore)
```

Comment:
Gradle's typed project accessors are the equivalent of Version Catalogs but for project dependencies.